src/cdk/layout/breakpoints.ts

```diff
@@ -8,6 +8,12 @@
 // PascalCase is being used as Breakpoints is used like an enum.
 // tslint:disable-next-line:variable-name
 export const Breakpoints = {
+  WindowXSmall: '(max-width: 599px)',
+  WindowSmall: '(min-width: 600px) and (max-width: 959px)',
+  WindowMedium: '(min-width: 960px) and (max-width: 1279px)',
+  WindowLarge: '(min-width: 1280px) and (max-width: 1919px)',
+  WindowXLarge: '(min-width: 1920px)',
```

Comment:
@josephperrott I looked at [the spec](https://material.io/guidelines/layout/responsive-ui.html) and it's not clear to me what "Window" even means in that table. Is it supposed be be a "desktop browser window"?

Reply:
I believe so.  That mostly is coming from listing breakpoints "across desktop, mobile and tablet".  And mobile (written elsewhere as handset) and tablet are referenced so the others must be desktop?

Reply:
Would it make sense to make these `DesktopSmall`, `DesktopMedium`, etc.?

It would also be nice to confirm with the Material Design team.

Reply:
After some discussion, @jelbourn and I determined that we think the best option would be to drop the prefix of the sizes entirely.

```
XSmall: '(max-width: 599px)',
Small: '(min-width: 600px) and (max-width: 959px)',
Medium: '(min-width: 960px) and (max-width: 1279px)',
Large: '(min-width: 1280px) and (max-width: 1919px)',
XLarge: '(min-width: 1920px)',
```

Reply:
Done